### DISCUSSION TITLE
Usage within vuepress

### DISCUSSION THREAD

**Discussion by pdnpa** [chartjs/chartjs-plugin-datalabels#231] [Q&A] [answered]
Hi,
I am using [vuepress-plugin-chart](https://www.npmjs.com/package/vuepress-plugin-chart) for using Chart.Js in Vuepress. I can't work out how to get this plugin to integrate - i.e. how can I load/import it as a dependency and register it with chart.js?

Does anyone have any working versions of chart.js inside vuepress using this (or any other) chart.js plugins?

Thanks for any help.

**Comment by simonbrunel** [comment, marked as answer]
Chart.js does **not** maintain `vuepress-plugin-chart`, so you should ask your questions to the developer of this vuepress plugin instead. Since I'm also using VuePress for this project, I can help on this request but if you need extra support, please [use their repository](https://github.com/Renovamen/vuepress-theme-gungnir/issues).

You can import and register Chart.js plugins in the [`.vuepress/enhanceApp.js` file](https://vuepress.vuejs.org/guide/basic-config.html#app-level-enhancements) of your documentation:

```js
// .vuepress/enhanceApp.js
import {Chart} from 'chart.js';
import ChartDataLabels from 'chartjs-plugin-datalabels';

export default () => {
  Chart.plugins.register(ChartDataLabels);
};
```

**Reply by pdnpa** [reply]
Thank you so much for this.
My problem was that I didn't know which element this question really related to - vuepress, chart.js, vuepress-plugin-chart or chartjs-plugin-datalabels - so thought I'd start asking somewhere and hope for someone helpful to point me in the right direction, which you have.

Your answer worked as expected.

So, many thanks again.
